uw-frame-components/css/angular.less

```diff
@@ -1,6 +1,6 @@
-@import "../bower_components/normalize-less/normalize.less";
-@import "../../bower_components/font-awesome/css/font-awesome.css";
-@import "../bower_components/bootstrap/less/bootstrap.less";
+@import "https://raw.githubusercontent.com/additiveinverse/normalize.less/master/normalize.less";
```

Comment:
K, I confirmed that bower was pulling from master, and probably intentionally, since the last tag (the only tag), v0.1, contains a much older version, 2.1.3.  Basically, this looks like an abandoned fork with weird versioning.
I fixed the URL to the current (two year old) revision so that the build is deterministic.